desktop/core/src/desktop/js/sql/reference/flink/udfReference.ts

```diff
@@ -252,7 +252,7 @@ const DATE_FUNCTIONS: UdfCategoryFunctions = {
   }
 };
 
-const AUXILIARY_FUNCTIONS: UdfCategoryFunctions = {
+const WINDOW_FUNCTIONS: UdfCategoryFunctions = {
```

Comment:
Group Window?

https://ci.apache.org/projects/flink/flink-docs-stable/dev/table/sql/queries.html#group-windows

(as Window Function officially refers to https://mode.com/sql-tutorial/sql-window-functions/)
(and in Hive/Impala as 'Analytic')

Reply:
Correcting it .

Reply:
I have correct it to Group Windows Functions